.github/actions/ci-setup/action.yml

```diff
@@ -0,0 +1,15 @@
+name: CI setup
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+        cache: "yarn"
+
+    - name: Install Dependencies
+      run: yarn install --frozen-lockfile --ignore-scripts
+      shell: bash
```

Comment:
Isn't bash default? Probably no need to specify

Reply:
Maybe something has changed but I had to add it explicitly in the past:
https://github.com/statelyai/xstate/compare/b8b4a79a7e53b2f79b102f9f12de16a72b728f02..f5485bdea0c4d9d257fd04c0fe4907720eb7e8af